components/StatBox/StatBox.tsx

```diff
@@ -11,12 +11,14 @@ type StatBoxProps = {
 	title: ReactNode;
 	value: ReactNode;
 	size?: Size;
+	children?: ReactNode;
 };
 
-const StatBox: FC<StatBoxProps> = ({ title, value, size = 'md', ...rest }) => (
+const StatBox: FC<StatBoxProps> = ({ title, value, size = 'md', children, ...rest }) => (
 	<Box {...rest} size={size}>
 		<Title className="title">{title}</Title>
 		<Value className="value">{value}</Value>
+		{children}
```

Comment:
I decided to pass `children` here to maximum flexibility... just in case we want to render something completely different after the title/value.